BLOCKFILE

```diff
@@ -56,7 +56,6 @@ hw/ip/prim_xilinx/rtl/*
 hw/ip/prim_xilinx_ultrascale/rtl/*
 hw/ip/pwm/rtl/*
 hw/ip/rom_ctrl/rtl/*
-hw/ip/rv_core_ibex/rtl/*
```

Comment:
I think we want to keep this line, as `rv_core_ibex_pkg.sv` still lives in this directory.

Reply:
Makes sense. Added back.